### PR TITLE
docs: add obsolete note to README.adoc

### PR DESCRIPTION
**Pull request by CAMOBAP**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,3 +1,29 @@
+[NOTE]
+====
+:bangbang: This repository is *obsolete* and no longer actively maintained.
+
+If you want to install the **Microsoft Vista Fonts**, we recommend using the `fontist` gem, which automates the installation of these fonts.
+
```
Comment (ronaldtse) [inline README.adoc:6]: Can we point people to the official fontist website at https://fontist.org ? Thanks.

File: README.adoc [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+----
+
+. Use `fontist` to install the Constantia font from Microsoft Vista fonts collection:
++
+[source, bash]
+----
+fontist install Constantia
+----
+
+For more information about the `fontist` gem, please visit the official repository at https://github.com/fontist/fontist.
```
Comment (CAMOBAP) [inline README.adoc:23]:
```suggestion
For more information about the `fontist` gem, please visit the official site at https://fontist.org/ .
```

File: README.adoc (hunk 1 above)
Comment (CAMOBAP) [reply README.adoc:6]: Done below https://github.com/fontist/vista-fonts-installer/pull/16/files#diff-1194561cbea346acb1bea37d81fdf5c70def982dcbb0c64eca2a96bc51941124R23

**Comment by ronaldtse** [comment]
Thank you @CAMOBAP !

**Pull request merged**